client/src/components/typing/long/List/index.tsx

```diff
@@ -76,14 +100,13 @@ export default function PracticeLongList({ currentPage, totalPage, typingList }:
               {typingList.map(({ language, thumbnail, title, totalPage, typingId, viewCount }: LongTypingItem) => (
                 <Tr key={typingId}>
```

Comment:
```suggestion
                <Tr key={typingId} title={thumbnail}>
```

이렇게 title 속성 넣어도 같은 기능을 하긴 해요. 하지만 스타일이나 기타 세부사항을 위해서라면 지금 하신 방식도 좋아보입니다!

Reply:
와 이런 신기한 기능이 있었네요..? 이걸 알았다면 title로 썸네일 넣고 스타일을 바꿀 수 있는지 찾아봤을것 같아요. 꿀팁 감사합니다!